core/store/src/main/java/mil/nga/giat/geowave/core/store/operations/remote/RemoveIndexCommand.java

```diff
@@ -44,6 +49,7 @@ public void execute(
 
 		throw new UnsupportedOperationException(
```

Comment:
I know this isn't particularly associated with the REST project, but `RemoteOperationsProvider` should not list `RemoveIndexCommand` merely to throw an UnsupportedOperationException (I just now realized this is happening). As long as we're in this code we should remove RemoveIndexCommand from RemoteOperationsProvider.